sass/main.scss

```diff
@@ -82,6 +82,7 @@ img {
 }
 
 .slide_dosomething {
```

Comment:
I can do another PR and rename everything to the same convention you guys are using :+1: